src/plasma/plasma_manager.cc

```diff
@@ -235,6 +235,11 @@ struct PlasmaManagerState {
   /** The time (in milliseconds since the Unix epoch) when the most recent
    *  heartbeat was sent. */
   int64_t previous_heartbeat_time;
+  /**
+   * Objects are mapped to the time, in milliseconds, they were received. Objects
+   * are removed from this map after they are added to local_available_objects.
+   */
```

Comment:
Can you elaborate a bit? When precisely are the objects added to this map? Is it when the transfer starts or the transfer finishes? What is the "time they were received"? Is it when the transfer started or completed?

Reply:
Will clarify in documentation.